### PR TITLE
Added metrics for disk attach/detach failure

### PR DESCRIPTION
**Pull request by alimez**
Basic metrics to track non existing disk paths and failed local unmount attempts

### CODE REVIEW

File: turbinia/processors/google_cloud.py [hunk 1]
```diff
@@ -25,13 +25,18 @@
...
+from prometheus_client import Gauge
 from turbinia import config
 from turbinia import TurbiniaException
 
 log = logging.getLogger('turbinia')
 
 RETRY_MAX = 10
 
+turbinia_nonexisting_disk_path = Gauge(
+    'turbinia_nonexisting__disk_path',
```
Comment (aarontp) [inline turbinia/processors/google_cloud.py:37]: Looks like an extra underscore here?

```suggestion
    'turbinia_nonexisting_disk_path',
```

File: turbinia/processors/mount_local.py [hunk 2]
```diff
@@ -21,14 +21,17 @@
...
-
+from prometheus_client import Gauge
 from turbinia import config
 from turbinia import TurbiniaException
 
 log = logging.getLogger('turbinia')
 
 RETRY_MAX = 10
 
+turbinia_failed_detach_disk = Gauge(
```
Comment (aarontp) [inline turbinia/processors/mount_local.py:32]: Can we also add these to the monitoring dashboards?

**Review by aarontp — COMMENTED** [review]
Thanks!  Should we add the loop device failures into this PR, or just do it in a separate one?

File: turbinia/processors/mount_local.py (hunk 2 above)
Comment (alimez) [reply turbinia/processors/mount_local.py:32]: Yes but they need to be done on the dashboard itself

**Comment by alimez** [comment]
> Thanks! Should we add the loop device failures into this PR, or just do it in a separate one?

Changes in mount_local.py file are for loop device failures

File: turbinia/processors/mount_local.py (hunk 2 above)
Comment (aarontp) [reply turbinia/processors/mount_local.py]: We talked out of band, and alimez is going to add these to the dashboard directly and then export and sync the resulting .json file into the repo.

File: turbinia/processors/google_cloud.py [hunk 3]
```diff
@@ -34,8 +34,7 @@
 RETRY_MAX = 10
 
 turbinia_nonexisting_disk_path = Gauge(
-    'turbinia_nonexisting__disk_path',
-    'Total number of non existing disk paths')
+    'turbinia_nonexisting_disk_path', 'Total number of non existing disk paths')
```
Comment (aarontp) [inline turbinia/processors/google_cloud.py:37]: Small thing (sorry I didn't notice this the first time), but we could probably clarify this a bit by adding something like "... after attempts to attach" to the description.

File: turbinia/processors/mount_local.py [hunk 4]
```diff
@@ -29,8 +29,9 @@
 
 RETRY_MAX = 10
 
-turbinia_failed_detach_disk = Gauge(
-    'turbinia_failed_detach_disk', 'Total number of disks failed to detach')
+turbinia_failed_loop_device_detach = Gauge(
+    'turbinia_failed_loop_device_detach',
+    'Total number of disks failed to detach')
```
Comment (aarontp) [inline turbinia/processors/mount_local.py:34]: s/disks/loop devices/ ?

**Review by aarontp — APPROVED** [review]
LG, just a couple small nits (sorry that I didn't pick those up the first round)

File: turbinia/processors/mount_local.py (hunk 4 above)
Comment (alimez) [reply turbinia/processors/mount_local.py]: fixed

File: turbinia/processors/google_cloud.py (hunk 3 above)
Comment (alimez) [reply turbinia/processors/google_cloud.py]: fixed

**Comment by aarontp** [comment]
After talking out of band, we decided to merge this now so we can push a new release and then the dashboard changes can come later.

**Pull request merged**